src/services/etherspot.js

```diff
@@ -698,6 +698,35 @@ export class EtherspotService {
       return [];
     }
   }
+
+  getContract<T>(
```

Comment:
Should we call this `registerContract` instead?

Reply:
I think no because the name of Etherspot's `registerContract` is used to register and return while we the returned contract and we don't care about the register part. The flow is to get contract instance, not to register it, however, `registerContract` is the only way to get the contract instance.